Build is broken

Example build: https://travis-ci.org/s12v/awsbeats/builds/382528626

Error:
```
Building filebeats:v6.2.4...
error: pathspec 'vv6.2.4' did not match any file(s) known to git.
make: *** [beats] Error 1
```

`vv6.2.4` doesn't look good, something is wrong in Makefile

--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 GO_VERSION=$(shell go version | cut -d ' ' -f 3 | sed -e 's/ /-/g' | sed -e 's/\//-/g' | sed -e 's/^go//g')
 GO_PLATFORM ?= $(go version | cut -d ' ' -f 4 | sed -e 's/ /-/g' | sed -e 's/\//-/g')
 BEATS_VERSION ?= "master"
-BEATS_TAG ?= $(shell echo ${BEATS_VERSION} | sed 's/\([[:digit:]]*\(\.[[:digit:]]*\)\)/v\1/')
+BEATS_TAG ?= $(shell echo ${BEATS_VERSION} | sed 's/[^[:digit:]]*\([[:digit:]]*\(\.[[:digit:]]*\)\)/v\1/')
 AWSBEATS_VERSION ?= "1-snapshot"
 DOCKER_IMAGE ?= s12v/awsbeats
 DOCKER_TAG ?= canary

